### ISSUE TITLE
umask fix is incomplete

### ISSUE THREAD

**Issue by ebfe** [voidlinux/xbps#213]
#108 fixed it for the create case, however the issue still exists when updating the pkgdb and a similar problem exists with the `/var/db/xbps/<repourl>/<arch>-repodata` files.

repro:

```
$ umask 0027
$ sudo xbps-install -S
# repodata perms flip
$ sudo xbps-install -S <some package>
# pkgdb perms flip
```

**Comment by emdete** [comment]
this seems to be a more generic problem. i suddenly couldn't start epiphany and meld because /usr/share/glib-2.0/schemas/gschemas.compiled got permission -rw-r-----. this seems to be related with a umask 027.

**Comment by xtraeme** [comment]
Fixed via #284
